apps/web/pages/signup.tsx

```diff
@@ -276,25 +268,29 @@ export default function Signup({
                 await signUp(updatedValues);
               }}>
               {/* Username */}
-              <UsernameField
-                label={t("username")}
-                username={watch("username")}
-                premium={premiumUsername}
-                usernameTaken={usernameTaken}
-                setUsernameTaken={(value) => setUsernameTaken(value)}
-                data-testid="signup-usernamefield"
-                setPremium={(value) => setPremiumUsername(value)}
-                addOnLeading={
-                  orgSlug
-                    ? `${getOrgFullOrigin(orgSlug, { protocol: true })}/`
-                    : `${process.env.NEXT_PUBLIC_WEBSITE_URL}/`
-                }
-              />
+              {!isOrgInviteByLink ? (
```

Comment:
Don't show the username field if it's invite link. Username would be derived from email automatically. This is same as earlier implementation before regression.